parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -624,18 +698,14 @@ impl pallet_collator_selection::Config for Runtime {
 	type WeightInfo = weights::pallet_collator_selection::WeightInfo<Runtime>;
 }
 
-impl pallet_asset_tx_payment::Config for Runtime {
+impl pallet_asset_conversion_tx_payment::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type Fungibles = Assets;
-	type OnChargeAssetTransaction = pallet_asset_tx_payment::FungiblesAdapter<
-		pallet_assets::BalanceToAssetBalance<
-			Balances,
-			Runtime,
-			ConvertInto,
-			TrustBackedAssetsInstance,
-		>,
-		AssetsToBlockAuthor<Runtime, TrustBackedAssetsInstance>,
+	type Fungibles = LocalAndForeignAssets<
+		Assets,
+		AssetIdForTrustBackedAssetsConvert<TrustBackedAssetsPalletLocation>,
+		ForeignAssets,
 	>;
+	type OnChargeAssetTransaction = AssetConversionAdapter<Balances, AssetConversion>;
```

Comment:
This `AssetConversionAdapter` doesn't seem to handle what to actually do with the fees (a la `AssetsToBlockAuthor`). Where does that get handled? This may have been a mistake in the Westend configuration.

Reply:
It seems that the fees might be handled by line 238 of this file.

The OnChargeTransaction of `pallet_transaction_payment::Config`, it has a `DealWithFees` struct that implements `OnUnbalanced`, however I'm not sure if it's the same as `AssetsToBlockAuthor` (is StakingPot similar? probably deferred for later payouts)

Reply:
Yeah right, fees are handled by `pallet_transaction_payment`